boards/cc2650stk/Makefile.include

```diff
@@ -0,0 +1,21 @@
+export CPU        = cc26x0
+export CPU_MODEL ?= cc26x0f128
+
+ifeq (,$(UNIFLASH_PATH))
+$(error UNIFLASH_PATH not set)
+endif
+
+# set default port depending on operating system
+PORT_LINUX ?= /dev/ttyACM0
+PORT_DARWIN ?= $(shell ls -1 /dev/tty.SLAB_USBtoUART* | head -n 1)
```

Comment:
use this instead: `PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))`

-> all these occurrences were changes to this style lately...